emukit/core/parameter_space.py

```diff
@@ -4,12 +4,18 @@
 import GPyOpt
 
 from . import ContinuousParameter
+from .discrete_parameter import DiscreteParameter, InformationSourceParameter
 
 
 class ParameterSpace(object):
```

Comment:
this class is lacking a dockstring.

Reply:
It is but I didn't add this class so I'll leave this for a different PR

Reply:
makes sense.